python/test/unit/language/test_core.py

```diff
@@ -5537,35 +5453,10 @@ def _kernel(dst):
         pass
 
     with pytest.raises(AssertionError, match='must be a power of 2'):
-        _kernel[(1, )](dst=dst, num_warps=3)
-    _kernel[(1, )](dst=dst, num_warps=1)
-    _kernel[(1, )](dst=dst, num_warps=2)
-    _kernel[(1, )](dst=dst, num_warps=4)
-
-
-@pytest.mark.interpreter
-@pytest.mark.parametrize("func_str", ['sqrt', 'rsqrt', 'exp', 'exp2', 'log', 'log2', 'sin', 'cos'])
-def test_unary_math(func_str, device):
```

Comment:
why kill this one?

Reply:
It's a duplicate test with `test_math_op`